src/pdf.js

```diff
@@ -32,24 +32,28 @@ let pdfjsDisplayAPICompatibility = require('./display/api_compatibility.js');
 
 if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
   const isNodeJS = require('./shared/is_node.js');
+  const isFetchSupported = require('./shared/is_fetch_supported.js');
```

Comment:
If we move this up, we probably don't have to repeat this below.

Reply:
This was done purposely, to avoid loading this in `MOZCENTRAL` builds.